pkg/scheduler/eventhandlers_test.go

```diff
@@ -443,3 +452,71 @@ func TestAddAllEventHandlers(t *testing.T) {
 		})
 	}
 }
+
+func TestAdmissionCheck(t *testing.T) {
+	nodeaffinityError := AdmissionResult{Name: nodeaffinity.Name, Reason: nodeaffinity.ErrReasonPod}
+	nodenameError := AdmissionResult{Name: nodename.Name, Reason: nodename.ErrReason}
+	nodeportsError := AdmissionResult{Name: nodeports.Name, Reason: nodeports.ErrReason}
+	podOverheadError := AdmissionResult{InsufficientResources: &noderesources.InsufficientResource{ResourceName: v1.ResourceCPU, Reason: "Insufficient cpu", Requested: 2000, Used: 7000, Capacity: 8000}}
+	cpu := map[v1.ResourceName]string{v1.ResourceCPU: "8"}
+	tests := []struct {
+		name                 string
+		nodeFunc             func() *v1.Node
+		existingPods         []*v1.Pod
+		pod                  *v1.Pod
+		flags                []bool
```

Comment:
It looks like this is always `{true, false}`, so you can just make it part of the test code instead of the test table:

```golang
for i := range []bool{true, false}
```

Reply:
fine